wcfsetup/install/files/style/font-awesome/000-woltlab.scss

```diff
@@ -0,0 +1,2141 @@
+/* Variables */
+$fa-font-path           : "../font";
+
+/* Imports */
+@import 'functions';
+@import 'variables';
+@import 'mixins';
+
+/* Fix for division:
+ * https://github.com/scssphp/scssphp/issues/504
+ */
+@mixin fa-size ($font-size) {
+  font-size: (($font-size / $fa-size-scale-base)) * 1em; // converts step in sizing scale into an em-based value that's relative to the scale's base
+  line-height: ((1 / $font-size)) * 1em; // sets the line-height of the icon back to that of it's parent
+  vertical-align: (((6 / $font-size)) - ((3 / 8))) * 1em; // vertically centers the icon taking into account the surrounding text's descender
+}
+$fa-fw-width            : ((20em / 16));
+$fa-li-margin           : ($fa-li-width * divide(5 / 4)) !default;
```

Comment:
This whole block should be adjusted to use the workaround recommended by scssphp in the linked issue.